modules/auxiliary/scanner/mssql/mssql_hashdump.rb

```diff
@@ -114,7 +114,7 @@ def report_hashes(mssql_hashes, version_year)
 
     service_data = {
         address: ::Rex::Socket.getaddress(rhost,true),
-        port: rport,
+        port: datastore['RPORT'],
```

Comment:
I think this has snuck in here accidentally, it looks like a rebase issue maybe?

Reminds me of this thread: https://github.com/rapid7/metasploit-framework/pull/18696#discussion_r1471957772